package.json

```diff
@@ -2,6 +2,11 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
+  "engines": {
+    "node": "13.12.0",
+    "npm": "6.14.4",
+    "yarn": "1.22.4"
+  },
```

Comment:
This was used for Heroku deployment, but did not work at all. If this is not needed for the dashboard, I will delete it before merge.